tests/Functional/ResultMetadataTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional;
+
+use Doctrine\DBAL\Exception\InvalidColumnIndex;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+
+class ResultMetadataTest extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        if (TestUtil::isDriverOneOf('pdo_oci')) {
+            self::markTestSkipped("PDO_OCI doesn't support binding binary values");
+        }
```

Comment:
I don't understand why the fact that we cannot bind binary values prevents us from testing fetching column names. 😕

Reply:
bad copy paste